doc/source/cluster/running-applications/job-submission/index.md

```diff
@@ -1,29 +1,40 @@
 (jobs-overview)=
 
-# Ray Jobs
+# Ray Jobs Overview
 
 Once you have deployed a Ray cluster (on [VMs](vm-cluster-quick-start) or [Kubernetes](kuberay-quickstart)), you are ready to run a Ray application!
 
-The recommended way to run a Ray application on a Ray Cluster is to use *Ray Jobs*.
-Ray Jobs allow you to submit locally developed applications to a remote Ray Cluster for execution.
+## Ray Jobs API
+
+The recommended way to run a job on a Ray cluster is to use the *Ray Jobs API*, which consists of a CLI tool, Python SDK, and a REST API.
+
+The Ray Jobs API allows you to submit locally developed applications to a remote Ray Cluster for execution.
 It simplifies the experience of packaging, deploying, and managing a Ray application.
 
-A Ray Job consists of:
-1. An entrypoint command, like `python my_script.py`.
+A submission to the Ray Jobs API consists of:
+
+1. An entrypoint command, like `python my_script.py`, and
 2. A [runtime environment](runtime-environments), which specifies the application's file and package dependencies.
 
-A Ray Job can be submitted by a remote client that lives outside of the Ray Cluster.
+A job can be submitted by a remote client that lives outside of the Ray Cluster.
 We will show this workflow in the following user guides.
 
-After a Ray Job is submitted, it runs once to completion or failure, regardless of the original submitter's connectivity.
+After a job is submitted, it runs once to completion or failure, regardless of the original submitter's connectivity.
 Retries or different runs with different parameters should be handled by the submitter.
 Jobs are bound to the lifetime of a Ray cluster, so if the cluster goes down, all running jobs on that cluster will be terminated.
 
-To get started with Ray Jobs, check out the [quickstart](jobs-quickstart) guide, which walks you through the CLI tools for submitting and interacting with a Ray Job.
+To get started with the Ray Jobs API, check out the [quickstart](jobs-quickstart) guide, which walks you through the CLI tools for submitting and interacting with a Ray Job.
 This is suitable for any client that can communicate over HTTP to the Ray Cluster.
-If needed, Ray Jobs also provides APIs for [programmatic job submission](ray-job-sdk) and [job submission using REST](ray-job-rest-api).
+If needed, the Ray Jobs API also provides APIs for [programmatic job submission](ray-job-sdk) and [job submission using REST](ray-job-rest-api).
+
+## Running Jobs Interactively
+
+If you would like to run an application *interactively* and see the output in real time (for example, during development or debugging), you can:
+
+- Run your script directly on a cluster node (e.g. after SSHing into the node), or
```

Comment:
Is there any doc that explains how to do this? Maybe we should have a link

Reply:
Ping on this.

Reply:
I didn't find any such doc.  I'll link to `ray attach` since it seems like the closest thing.

Reply:
`ray attach` won't work for KubeRay right?
Could we actually write a section about it? For people who want to build a platform/service on top of Ray, this is a way people should consider instead of ray client right?

Reply:
@DmitriGekhtman do you know if `ray attach` works for Kuberay, or what the recommended approach is there? (Is it some form of `kubectl exec`)?

@scottsun94 I think users who are running a cluster probably already have a way to run commands on their cluster, almost as a prerequisite (in this case the command we're telling them to run is `python my_ray_script.py`).  So my impression is we don't need a whole section for this, and can just mention SSH or `kubectl` in passing. What do you think?

For people building on top of Ray, we would recommend using the Jobs API or directly running driver scripts.

Reply:
Yeah. Probably a short sentence that covers both VM and KubeRay should be fine for now.